models/bancor/ethereum/bancor_ethereum_trades.sql

```diff
@@ -0,0 +1,236 @@
+
+WITH conversions AS (
+    SELECT  t.evt_block_time,
+            t._trader,
+            t._toAmount,
+            t._fromAmount,
+            t._toToken,
+            t._fromToken,
+            t.contract_address,
+            t.evt_tx_hash,
+            t.evt_index
+    FROM {{ source('bancornetwork_ethereum', 'BancorNetwork_v6_evt_Conversion') }} t
+    {% if is_incremental() %}
+    WHERE t.evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+    {% if not is_incremental() %}
+    WHERE t.evt_block_time >= '{{project_start_date}}'
+    {% endif %}
+
+    UNION ALL
+
+    SELECT  t.evt_block_time,
+            t._trader,
+            t._toAmount,
+            t._fromAmount,
+            t._toToken,
+            t._fromToken,
+            t.contract_address,
+            t.evt_tx_hash,
+            t.evt_index
+    FROM {{ source('bancornetwork_ethereum', 'BancorNetwork_v7_evt_Conversion') }} t
+    {% if is_incremental() %}
+    WHERE t.evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+    {% if not is_incremental() %}
+    WHERE t.evt_block_time >= '{{project_start_date}}'
+    {% endif %}
+
+    UNION ALL
+
+    SELECT  t.evt_block_time,
+            t._trader,
+            t._toAmount,
+            t._fromAmount,
+            t._toToken,
+            t._fromToken,
+            t.contract_address,
+            t.evt_tx_hash,
+            t.evt_index
+    FROM {{ source('bancornetwork_ethereum', 'BancorNetwork_v8_evt_Conversion') }} t
+    {% if is_incremental() %}
+    WHERE t.evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+    {% if not is_incremental() %}
+    WHERE t.evt_block_time >= '{{project_start_date}}'
+    {% endif %}
+
+    UNION ALL
+
+    SELECT  t.evt_block_time,
+            t._trader,
+            t._toAmount,
+            t._fromAmount,
+            t._toToken,
+            t._fromToken,
+            t.contract_address,
+            t.evt_tx_hash,
+            t.evt_index
+    FROM {{ source('bancornetwork_ethereum', 'BancorNetwork_v9_evt_Conversion') }} t
+    {% if is_incremental() %}
+    WHERE t.evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+    {% if not is_incremental() %}
+    WHERE t.evt_block_time >= '{{project_start_date}}'
+    {% endif %}
+
+    UNION ALL
+
+    SELECT  t.evt_block_time,
+            t._trader,
+            t._toAmount,
+            t._fromAmount,
+            t._toToken,
+            t._fromToken,
+            t.contract_address,
+            t.evt_tx_hash,
+            t.evt_index
+    FROM {{ source('bancornetwork_ethereum', 'BancorNetwork_v10_evt_Conversion') }} t
+    {% if is_incremental() %}
+    WHERE t.evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+    {% if not is_incremental() %}
+    WHERE t.evt_block_time >= '{{project_start_date}}'
+    {% endif %}
+),
+
+dexs AS (
+SELECT
+    '1' AS version,
```

Comment:
I use '1' as the default version here I am not sure if I am correct

Reply:
i think that is fair if there is no other reference in place to build off